libs/feature/search/src/lib/state/effects.ts

```diff
@@ -130,8 +131,17 @@ export class SearchEffects {
               return of([state, favorites, null])
             }
             return this.filterGeometry$.pipe(
+              tap((geom) => {
+                const isValid = validGeoJson(geom)
+                if (!isValid) {
+                  throw '\nFilter geometry is not a valid GeoJson'
+                }
+              }),
               map((geom) => [state, favorites, geom]),
-              catchError(() => of([state, favorites, null])) // silently opt out of spatial filter if an error happens
+              catchError((e) => {
+                console.warn('The filter geometry cannot be used', e)
```

Comment:
This is not silent any more since we're outputting something in the console now. I would remove this `console.warn` call and instead print to the console if the geojson is invalid.